packages/tizen_audio_manager/tizen/src/tizen_audio_manager_plugin.cc

```diff
@@ -89,180 +54,123 @@ class AudioManagerTizenPlugin : public flutter::Plugin {
   virtual ~AudioManagerTizenPlugin() { UnregisterObserver(); }
 
  private:
-  void HandleMethodCall(
-      const flutter::MethodCall<flutter::EncodableValue> &method_call,
-      MethodResultPtr result) {
-    LOG_INFO("method: %s", method_call.method_name().c_str());
-
-    const auto &arguments = *method_call.arguments();
+  void HandleMethodCall(const FlMethodCall &method_call,
+                        MethodResultPtr result) {
+    const std::string &method_name = method_call.method_name();
 
-    if (method_call.method_name().compare("getCurrentPlaybackType") == 0) {
-      GetCurrentPlaybackType(std::move(result));
-    } else if (method_call.method_name().compare("getMaxLevel") == 0) {
-      GetMaxLevel(arguments, std::move(result));
-    } else if (method_call.method_name().compare("setLevel") == 0) {
-      SetLevel(arguments, std::move(result));
-    } else if (method_call.method_name().compare("getLevel") == 0) {
-      GetLevel(arguments, std::move(result));
+    if (result_) {
+      result_->Error("Already active", "Cancelled by a second request.");
```

Comment:
cancelled → canceled

Also only synchronous functions are used in the implementation so I doubt if this check and the extra variable (`result_`) are necessary.

Reply:
done